### DISCUSSION TITLE
where is delve installed?

### DISCUSSION THREAD

**Discussion by montanaviking** [go-delve/delve#2970] [Q&A]
Hi,
I am running Ubuntu 18.04 and I installed the latest Go compiler via snap:
$ sudo snap install go --classic
Then I attempted to install delve from Github:
$ go install github.com/go-delve/delve/cmd/dlv@latest
However, I cannot find the delve executable.
Also
$ go env GOBIN
returns nothing.
Any advice much appreciated.
Thanks, Phil

**Comment by aarzilli** [comment]
```
$ go help install
usage: go install [build flags] [packages]

Install compiles and installs the packages named by the import paths.

Executables are installed in the directory named by the GOBIN environment
variable, which defaults to $GOPATH/bin or $HOME/go/bin if the GOPATH
environment variable is not set. Executables in $GOROOT
are installed in $GOROOT/bin or $GOTOOLDIR instead of $GOBIN.
```

**Comment by montanaviking** [comment]
Thanks so much! I do see now where Delve (dlv) executable was installed. It
was in $HOME/go/bin.
Thanks,
Phil
